webapps/src/main/webapp/vue-app/taskCommentsDrawer/components/TaskCommentEditor.vue

```diff
@@ -119,8 +119,11 @@ export default {
   },
   mounted() {
     this.$root.$on('task-comment-created', () => {
-      this.inputVal = '';
-      this.editorReady = false;
+      this.reset();
+    });
+
+    document.addEventListener('Task-comments-drawer-closed', () => {
```

Comment:
try to add `beforeDestroy` please + `removeEventListener` since this is a component that can be deleted from UI.

Reply:
done!